Trent edits 20230927 (#82)
jessegmeyerlab/proteomics-tutorial: content/12.Data-Acquisition.md

This was shown to increase proteome converage. 

#### DDA methods for modifications

> Resonant CID [@DOI:https://doi.org/10.1074/mcp.m111.009910] and beam-type HCD [@DOI:https://doi.org/10.1021/pr100637q] are the most popular methods for unmodified and modified peptides due to their speed, accessibility, and efficiency. 

Reviewer: ```suggestion
Resonant CID [@DOI:10.1074/mcp.m111.009910] and beam-type HCD [@DOI:10.1021/pr100637q] are the most popular methods for unmodified and modified peptides due to their speed, accessibility, and efficiency. 
```

> Optimal dissociation methods for phosphopeptide identification and phosphosite localization include ExD-based methods, discussed later in more depth [@DOI:https://doi.org/10.1021/acs.analchem.7b00212; https://doi.org/10.1021/pr301130k]. 

Reviewer: ```suggestion
Due to the weak phosphoester bond relative to the peptide backbone, resonant CID usually produces spectra that are dominated by only the neutral loss of the phosphate. 
For this reason, the optimal dissociation methods for phosphopeptide identification and phosphosite localization include HCD or ExD-based methods, discussed later in more depth [@DOI:10.1021/acs.analchem.7b00212; @DOI:10.1021/pr301130k]. 
```

> ExD methods generate phosphopeptide MS/MS spectra with many c- and z•-type fragment ions for peptide sequencing and localization of labile phosphate modifications, typically disrupted with CID [@DOI:https://doi.org/10.1073/pnas.0402700101]. 

Reviewer: ```suggestion
ExD methods generate phosphopeptide MS/MS spectra with many c- and z•-type fragment ions for peptide sequencing and localization of labile phosphate modifications, typically disrupted with CID [@DOI:10.1073/pnas.0402700101]. 
```

> Gas-phase phosphate rearrangement induced by collisional activation represents a glaring challenge for the field and several have explored site localization in the face of rearrangement [@DOI:https://doi.org/10.1021/ac801768s; https://doi.org/10.1002/pmic.201200384; https://doi.org/10.1074/mcp.m900619-mcp200].

Reviewer: ```suggestion
Gas-phase phosphate rearrangement induced by collisional activation represents a glaring challenge for the field and several have explored site localization in the face of rearrangement [@DOI:10.1021/ac801768s; @DOI:10.1002/pmic.201200384; @DOI:10.1074/mcp.m900619-mcp200].
```

Advanced data acquisition schemes trigger predetermined MS/MS events when a specific fragment ion or neutral loss is detected in a spectrum. 

> Certain decision-tree strategies have arisen to increase data acquisition efficiency, including pseudo-MS3 scans which are triggered on detection of phosphate losses [@DOI:https://doi.org/10.1021/ac0497104] and the use of site-specific x-type ions [@DOI:https://doi.org/10.1021/pr200154t]. 

Reviewer: ```suggestion
Certain decision-tree strategies have arisen to increase data acquisition efficiency, including pseudo-MS3 scans which are triggered on detection of phosphate losses [@DOI:10.1021/ac0497104] and the use of site-specific x-type ions [@DOI:10.1021/pr200154t]. 
```

For example, when linear ion traps were the main proteomics workhorses, resonant CID analysis of phosphopeptides would result in predominantly neutral loss of the phosphate with limited sequence ion information. 
To gain sequence ions in these experiments, instruments could be set to isolate a loss of 98 Thompsons for MS3 activation [@DOI:10.1073/pnas.0404720101; @DOI:10.1002/pmic.200800283].
The newer collisional dissociation technique HCD, or beam-type collisional activation, significantly improves the detection of peptide fragments with the phosphorylation intact on fragment ions, and thus, this neutral loss scanning technique is no longer common. 

> Recently developed approaches to phosphopeptide identification include DIA-based phosphoproteomics with Spectronaut [@DOI:https://doi.org/10.1126/scisignal.aaa3139; https://doi.org/10.1007/978-1-0716-1641-3_6], “plug-and-play” high-resolution MS [@DOI:https://doi.org/10.1038/nmeth.3811], SureQuant for phosphotyrosine [@DOI:https://doi.org/10.1158/0008-5472.can-20-3804], and FAIMS front-end separations which yield 15-20% more phosphosite identifications than non-FAIMS experiments [@DOI:https://doi.org/10.1021/acs.analchem.8b02233]. 

Reviewer: ```suggestion
Recently developed approaches to phosphopeptide identification include DIA-based phosphoproteomics with Spectronaut [@DOI:10.1126/scisignal.aaa3139; @DOI:10.1007/978-1-0716-1641-3_6], “plug-and-play” high-resolution MS [@DOI:10.1038/nmeth.3811], SureQuant for phosphotyrosine [@DOI:10.1158/0008-5472.can-20-3804], PIQED for direct identification and quantification of phosphorylation from DIA without a prior spectral library [@DOI:10.1038/nmeth.4334], and FAIMS front-end separations which yield 15-20% more phosphosite identifications than non-FAIMS experiments [@DOI:10.1021/acs.analchem.8b02233]. 
```

> For quantification of phosphoproteins, Hogrebe et al. investigated several of the most common strategies and concluded that TMT-based MS2 strategies may be the current best approach [@DOI:https://doi.org/10.1038/s41467-018-03309-6]. 

Reviewer: ```suggestion
For quantification of phosphoproteins, Hogrebe et al. investigated several of the most common strategies and concluded that TMT-based MS2 strategies may be the current best approach [@DOI:10.1038/s41467-018-03309-6]. 
```

> Additionally, while less commonly modified than serine and threonine, histidine [@DOI:https://doi.org/10.1038/nmeth.4580; https://doi.org/10.1021/ac0707838; https://doi.org/10.1038/s41592-022-01524-0], arginine [@DOI:https://doi.org/10.1002/pmic.201200240], and tyrosine [@DOI:https://doi.org/10.1073/pnas.2436191100; https://doi.org/10.1158/0008-5472.can-20-3804; https://doi.org/10.1016/j.cell.2006.09.026] phosphorylation also represent intriguing cell signaling biology. 

Reviewer: ```suggestion
Additionally, while less commonly modified than serine and threonine, histidine [@DOI:10.1038/nmeth.4580; @DOI:10.1021/ac0707838; @DOI:10.1038/s41592-022-01524-0], arginine [@DOI:10.1002/pmic.201200240], and tyrosine [@DOI:10.1073/pnas.2436191100; @DOI:10.1158/0008-5472.can-20-3804; @DOI:10.1016/j.cell.2006.09.026] phosphorylation also represent intriguing cell signaling biology. 
```
